deltachat-ios/Controller/MapViewController.swift

```diff
@@ -0,0 +1,33 @@
+import UIKit
+import WebKit
+import DcCore
+
+class MapViewController: WebxdcViewController {
+    private let isGlobalMap: Bool
+
+    init(dcContext: DcContext, chatId: Int) {
+        isGlobalMap = chatId == 0
```

Comment:
I think I asked this before, but what does `chatId == 0` mean again? Would it make sense to put this somewhere, i.e. a computed property/variable/... with a fitting name?

Reply:
it usually means "no filter, show everything". this pattern us used at many places, eg. also for global gallery etc. but maybe changing the parameter to `chatIdFilter: Int? = nil` makes it more obvious